upmc-enterprises/cda2fhir
# Feature/upmcfhir 334 jf unmapped fields

#### What does this PR do?
Compares the jolt report `.csv` files to the original `.xml` to find fields to leave behind the fields that we _have not_ mapped. 

#### Related JIRA tickets:
UPMCFHIR-334.

#### How should this be manually tested?

1. Copy the results of the .xml files under `/cda2fhir/src/test/resources/unmapped/` These are the processed `.xml` files with the mapped nodes already removed.
2. Go to `/cda2fhir/src/test/java/tr/com/srdc/cda2fhir/jolt/report/UnmappedFieldTest.java`.
3. Comment out the function calls to `generateFilteredNodeList` for each of the tests. Then run the tests.
4. Go back to `/cda2fhir/src/test/resources/unmapped/` and compare each file to the original files you copied in step 1. The first column of the `.csv` files located at `/cda2fhir/src/test/resources/gold/jolt-report` will tell you what should be parsed out. 

Note that any parent node that would otherwise be empty from the removal should be empty. 

#### Background/Context:

#### New JIRA tickets for clinical review required?
